src/answers-headless.ts

```diff
@@ -295,5 +296,31 @@ export default class AnswersHeadless {
     };
     this.stateManager.dispatchEvent('filters/toggleFacetOption', payload);
   }
+
+  addFilters(filterSetId: string, filters: SelectableFilter[]): void {
```

Comment:
I'm thinking that we should discuss the public interface of filters as a team and come to an agreement on it. Then we can present it to product, and once they have signed-off, then we come back to this PR to make updates